packages/shopping/src/application.ts

```diff
@@ -73,14 +72,12 @@ export class ShoppingApplication extends BootMixin(
   constructor(options?: ApplicationConfig) {
     super(options);
 
-    this.setUpBindings();
-
     // Bind authentication component related elements
     this.component(AuthenticationComponent);
+    this.component(JWTAuthenticationComponent);
     this.component(AuthorizationComponent);
 
-    // authentication
-    this.add(createBindingFromClass(JWTAuthenticationStrategy));
+    this.setUpBindings();
```

Comment:
Why do we need this?

Reply:
I see it is a method below.

Reply:
yeah we organize all bindings inside that function.